hono/src/schema.ts

```diff
@@ -81,3 +90,7 @@ export const itemsToOutfitsRelations = relations(itemsToOutfits, ({ one }) => ({
     references: [outfits.id],
   }),
 }))
+
```

Comment:
I'd add a comment here similar to the others in this file.

Reply:
Good idea.